src/components/Footer/styles.module.scss

```
@@ -0,0 +1,22 @@
+@import "../../reset.module.scss";
+@import "../../container.module.scss";
+
+.footer {
+  background: rgba(46, 80, 113, 1);
+  & > .container {
```

```suggestion
  > .container {
```
O `&` é desnecessário aqui, useo apenas quando quizer um seletor para items no mesmo nível.
ex: <input class="classA classB>